src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Area, Info {

```

Comment:
Don't begin class or method implementation with an empty line. The empty line after the class declaration (line 2) should be removed.